br/pkg/task/common.go

```diff
@@ -188,9 +188,10 @@ func DefineCommonFlags(flags *pflag.FlagSet) {
 
 	// Default concurrency is different for backup and restore.
 	// Leave it 0 and let them adjust the value.
-	flags.Uint32(flagConcurrency, 0, "The size of thread pool on each node that executes the task")
-	// It may confuse users , so just hide it.
-	_ = flags.MarkHidden(flagConcurrency)
+	flags.Uint32(flagConcurrency, 0, "The size of thread pool on br that executes the task, " +
+		"the task level of the backup and restore is different. " +
+		"so the default value must be different. if don't set. " +
+		"the default value of backup will adjust to 4, (restore is 128)")
```

Comment:
in this situation maybe it's better to just move this out of `DefineCommonFlags`, and use different wordings for restore and backup.

Reply:
OK, moved concurrency out of Common.